documentation/meeting-logs/2023-10-23.md

```diff
@@ -0,0 +1,109 @@
+# Standup Meeting 6
+
+2023-10-23
+Sprint 3
+
+Meeting duration: 2 hour
+
+## Attendees
+
+All group members
+
+## Notes
+
+Updates:
+
+- Dan
+  - Made two new endpoints for accessing it separately
+    - (<https://github.com/BarbzCodez/Spendr/issues/52>)
+  - presentation for friday
+- Victoria
+  - Cleaned up front end
+    - Made the colors to be by default on dark more
+    - Date added to expense dialog
+  - Made a Analytics UI
+- Ethan
+  - Finished up the prisma
+    - (<https://github.com/BarbzCodez/Spendr/issues/29>)
+  - fixed the middle ware logging bug pop up
+  - May have to change the testing because it cant go through the pipeline
+  - Changing to Morgan, dont have to test it because it is considered middle ware
+- Mark
+  - CD pipeline is completed, PR is up
+    - (<https://github.com/BarbzCodez/Spendr/issues/100>)
+  - Starting working on the budget api
+- Barbra
+  - Cleaned up front end
+    - Made the colors to be by default on dark more
+    - Date added to expense dialog
+    - (<https://github.com/BarbzCodez/Spendr/issues/23>)
+  - Made a Analytics UI
```

Comment:
Victoria and I have the same entry
```suggestion
- Victoria
  - Cleaned up front end
    - Made the colors to be by default on dark more
    - Date added to expense dialog
  - Made a Analytics UI
- Ethan
  - Finished up the prisma
    - (<https://github.com/BarbzCodez/Spendr/issues/29>)
  - fixed the middle ware logging bug pop up
  - May have to change the testing because it cant go through the pipeline
  - Changing to Morgan, dont have to test it because it is considered middle ware
- Mark
  - CD pipeline is completed, PR is up
    - (<https://github.com/BarbzCodez/Spendr/issues/100>)
  - Starting working on the budget api
- Barbara
  - Finished Budgets UI and middleware with the api, waiting for Delete and Edit api to be done
  - Bug fix: Hide the expense table
  - Cleaned off Documentation
```